### ISSUE TITLE
Skip-Thought Vectors: Vocabulary file for the decoder

### ISSUE THREAD

**Issue by abalkis** [tensorflow/models#4440]
### System information
- **What is the top-level directory of the model you are using**: N/A
- **Have I written custom code (as opposed to using a stock example script provided in TensorFlow)**: N/A
- **OS Platform and Distribution (e.g., Linux Ubuntu 16.04)**: N/A
- **TensorFlow installed from (source or binary)**: N/A
- **TensorFlow version (use command below)**: N/A
- **Bazel version (if compiling from source)**: N/A
- **CUDA/cuDNN version**: N/A
- **GPU model and memory**: N/A
- **Exact command to reproduce**: N/A

Hello,

I would like to use the pretrained decoder as a language model to generate sentences (as described in the original skip-thought paper). For this, we check the output of the softmax layer in the decoder, which is of size 20000 (i.e the number of words in the original vocabulary). The problem is that we do not have this vocabulary file which maps the softmax output to the actual word predicted. Could you please include that vocabulary file in the data?

Thank you!

**Comment by k-w-w** [comment]
FYI, in this case the top-level directory of the model would be research/skip_thoughts. It is helpful for us to quickly figure out and assign the model author to this problem if needed.

Taking a look at the readme, the `vocab.txt` file is generated by running the preprocess_dataset script. [This section described how to run it](https://github.com/tensorflow/models/tree/master/research/skip_thoughts#prepare-the-training-data).

**Comment by harshini-gadige** [comment]
Closing as this has been answered and in "awaiting response" status for more than a week.  Please add comments if any, we will reopen it. Thanks !
